### PR TITLE
OCLOMRS-467: Fix Retired count in the table of Concepts as well as the Dictionary Overview page

### PR DESCRIPTION
**Pull request by judeinno**
# JIRA TICKET NAME:
[Fix Retired count in the table of Concepts as well as the Dictionary Overview page](https://issues.openmrs.org/browse/OCLOMRS-467)

# Summary:
- Ensure that the number of retired concepts shown in on the Dictionary overview page.
- Matches the number of retired concepts shown in the table and ensure they add up.

### CODE REVIEW

File: src/tests/Signup/index.test.jsx [hunk 1]
```diff
@@ -28,4 +28,18 @@ describe('Sign up Component', () => {
     wrapper.find('#firstName').simulate('change', event);
     expect(spy).toHaveBeenCalledTimes(1);
   });
+
+  it('should handle text lastname input', () => {
+    const event = {
```
Comment (dkayiwa) [inline src/tests/Signup/index.test.jsx]: How is this related to the ticket?

Comment (judeinno) [reply src/tests/Signup/index.test.jsx]: The test coverage drop was arising from the Signup component so I worked on raising the test coverage.

Comment (dkayiwa) [reply src/tests/Signup/index.test.jsx]: Can you use new tickets and pull requests for cases of increasing coverage?

Comment (judeinno) [reply src/tests/Signup/index.test.jsx]: Alright, I will do that.

Comment (dkayiwa) [reply src/tests/Signup/index.test.jsx]: Remember to remove this change when you are done with that.

Comment (judeinno) [reply src/tests/Signup/index.test.jsx]: Alright.
